tests/Unit/Models/PostModelTest.php

```diff
@@ -182,28 +182,36 @@ public function testVoterRegActionStats()
     {
         $actionId = factory(Action::class)->create()->id;
         $schoolId = $this->faker->school_id;
+        $location = 'US-FL';
+
+        $group = factory(Group::class)->create([
+            'location' => $location,
+            'school_id' => $schoolId,
+        ]);
 
         factory(Post::class, 7)->states(['voter-reg', 'register-form'])->create([
             'action_id' => $actionId,
-            'school_id' => $schoolId,
+            'group_id' => $group->id,
```

Comment:
Ah so if a post's group has a `school_id`, the `school_id` is automatically saved on the post as well? (Is that how our OVRD group campaigns are functioning, hence these ActionStat leaderboards?)

Reply:
Exactly! Saving the `school_id` per `group_id` was added in #1064.